src/Controls/src/Core/Compatibility/Handlers/Shell/Android/ShellBottomNavViewAppearanceTracker.cs

```diff
@@ -142,15 +142,7 @@ ColorStateList MakeColorStateList(Color titleColor, Color disabledColor, Color u
 
 		static ColorStateList MakeColorStateList(int titleColorInt, int disabledColorInt, int defaultColor)
 		{
-			var states = new int[][] {
-				new int[] { -R.Attribute.StateEnabled },
-				new int[] {R.Attribute.StateChecked },
-				new int[] { }
-			};
```

Comment:
This `int[][]` was the same as `ColorStates.Switch`, so I just used `ColorStateListExtensions.CreateSwitch()`.